src/main/java/com/target/studentinfo/controller/StudentController.java

```diff
@@ -0,0 +1,59 @@
+package com.target.studentinfo.controller;
+
+import com.target.studentinfo.dto.request.StudentRequest;
+import com.target.studentinfo.dto.response.StudentResponse;
+import com.target.studentinfo.model.Student;
+import com.target.studentinfo.service.StudentService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/student_information_service/v1")
+public class StudentController {
+    private final StudentService studentService;
+
+    public StudentController(StudentService studentService) {
+        this.studentService = studentService;
+    }
+
+    @GetMapping("/students")
+    public List<Student> getAllStudentInfo(){
+
+        return studentService.getAllStudentInfo();
+    }
+
+    @GetMapping("/student/{id}")
```

Comment:
It should be `/students/{id}`. Please use `students` plural everywhere in the urls.